scripts/ci/pre_commit/sync_init_decorator.py

```diff
@@ -119,10 +119,6 @@ def _expr_to_ast_dump(expr: str) -> str:
     # Mapping of allowble Decorator type -> Class attribute type
     _expr_to_ast_dump("Collection[str] | None"): _expr_to_ast_dump("MutableSet[str]"),
     _expr_to_ast_dump("ParamsDict | dict[str, Any] | None"): _expr_to_ast_dump("ParamsDict"),
-    # TODO: This one is legacy access control. Remove it in 3.0. RemovedInAirflow3Warning
-    _expr_to_ast_dump(
-        "dict[str, dict[str, Collection[str]]] | dict[str, Collection[str]] | None"
-    ): _expr_to_ast_dump("dict[str, dict[str, Collection[str]]] | None"),
```

Comment:
@ashb Cleaning this makes static checks fail.

Is there a simple fix or shall I revert this change to make the PR green?

Reply:
Not really sure off the top of my head, sorry.

Reply:
Okay, then I revert this piece of cleanup, looks for me something open in TaskSDK that needs to be fixed.